docker/app/Dockerfile

```diff
@@ -12,6 +12,15 @@ RUN wget -q -O /tmp/shellcheck.tar.xz "https://storage.googleapis.com/shellcheck
   && rm /tmp/shellcheck.tar.xz \
   && shellcheck --version
 
+RUN wget -q -O get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
```

Comment:
For now, please stick to helm 2 (e.g. [v2.16.1](https://github.com/helm/helm/releases/tag/v2.16.1)) since we haven't updated our charts to helm 3 yet. We're tracking work to migrate to helm 3 in https://github.com/ascoderu/opwen-cloudserver/issues/283